about.html

```diff
@@ -4,15 +4,33 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="stylesheet.css">
-    <title>Document</title>
+    <script src="https://kit.fontawesome.com/8d3908a90c.js" crossorigin="anonymous"></script>
+    <title>Final Website</title>
 </head>
 <body>
-    <nav>
-        <ul>
-          <li><a href="index.html">Home</a></li>
-          <li><a href="about.html">About Us</a></li>
-          <li><a href="contact.html">Contact Us</a></li>
-        </ul>
-      </nav>
+  <nav>
+    <div class="navbar">
+      <a href="https://www.seasmoss.com/"><img class="logo" src="images.img/Original Logo.png" alt="Logo"></a>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About Us</a></li>
+        <li><a href="contact.html">Contact Us</a></li>
+      </ul>
+    </div>
+    </nav>
+      <h3>About Us</h3>
+      <p>
+        In todays society viruses, infections, and many other health problems have become a weapon of mass destruction amongst humanity. Here at Seá's Moss we believe health is wealth and we prioritize making the world a healthier place one jar at a time. Although we are not health physicians the benefits of sea moss has proved itself since 600bc. <br>
```

Comment:
Are we using prettier. It should automatically format for us. Don't look like it is. We might have to manually configure it. Remind me in a breakout room or you can google @kersea